crates/stackable-telemetry/src/tracing/settings/mod.rs

```diff
@@ -16,6 +16,11 @@ pub use otlp_log::*;
 pub mod otlp_trace;
 pub use otlp_trace::*;
 
+pub trait SettingsToggle {
+    fn is_enabled(&self) -> bool;
+    fn is_disabled(&self) -> bool;
```

Comment:
Do we really need both of these?

Reply:
technically no, but it does make the `assert!` look nicer (instead of `assert!(!...is_enabled())`.

Technically none of it is required, but it gets uglier to check the variant.

Reply:
What I will do is make is a default impl though.

Reply:
Disagree on the nicer, but we could at least provide `is_disabled()` as a default impl (especially since the concrete impls just delegate to it anyway).

Reply:
Ah, didn't see the followup message. :P